serverless/develop/serverless-using-brokers.adoc

```diff
@@ -1,30 +1,30 @@
 :_content-type: ASSEMBLY
 [id="serverless-using-brokers"]
-= Brokers
+= Creating brokers
 include::_attributes/common-attributes.adoc[]
 :context: serverless-using-brokers
 
 toc::[]
 
-include::snippets/serverless-brokers-intro.adoc[]
+Knative provides a default, channel-based broker implementation. This channel-based broker can be used for development and testing purposes, but does not provide adequate event delivery guarantees for production environments.
 
-include::modules/serverless-broker-types.adoc[leveloffset=+1]
+If your {ServerlessProductName} deployment is not configured to use Kafka broker as the default broker type, the channel-based broker is created when you use the default settings in the following procedures.
 
-[id="serverless-using-brokers-creating-brokers"]
-== Creating a broker that uses default settings
+If a cluster administrator has configured your {ServerlessProductName} deployment to use Kafka as the default broker type, creating a broker by using the default settings creates a Kafka-based broker.
 
-{ServerlessProductName} provides a `default` Knative broker that you can create by using the Knative (`kn`) CLI. You can also create the `default` broker by adding the `eventing.knative.dev/injection: enabled` annotation to a trigger, or by adding the `eventing.knative.dev/injection=enabled` label to a namespace.
+:FeatureName: Kafka broker
+include::snippets/technology-preview.adoc[leveloffset=+2]
```

Comment:
Should we still have this "Kafka broker is a Technology Preview feature only..." snippet?

Reply:
I would like to leave it in for now so we can merge this PR and remove them all in a later PR for 1.25.

Reply:
There will be multiple other changes to Kafka docs in the coming weeks before we mark everything as GA, we will do that at 1.25 release, but it shouldn't prevent us from merging PRs to add extra docs in the meantime for functionality in Kafka that already exists.